config/redirects.txt

```diff
@@ -13,3 +13,6 @@
 ^\/getting-started -> /raspberrypi/nodejs/getting-started
 
 ^\/configuration\/resin-base-images(\/|$) -> /runtime/resin-base-images
+
+# Renamed pages
+^\/deployment\/wifi\/?(.*)? -> /deployment/network/$1
```

Comment:
Very good point! How's this?  `^\/deployment\/wifi([\/]?|[\/]?(#.*)?)$ -> /deployment/network/$2`
Tested here: http://regexr.com/3e4m8 Let me know if you have any idea to simplify it....
If okay, I'll add a new commit. Cheers!